### PR TITLE
Exclude libaom for cloud gaming.

### PR DESCRIPTION
**Pull request by jianjunz**

### CODE REVIEW

**Review by taste1981 — COMMENTED** [review]
On server side, although encoder does not rely on libaom, for decoder, server implementation may either use customized decoder or use default decoder during its initialization. I just want to make sure  av1 still work without default libaom decoder? (With server not receiving any AV1 stream).

**Review by taste1981 — APPROVED** [review]
LGTM. Please confirm functionality before you merge it.

File: scripts/build-win.py [hunk 1]
```diff
@@ -83,6 +82,9 @@ def gngen(arch, sio_root, ffmpeg_root, ssl_root, msdk_root, quic_root, scheme, t
         gn_args.append('owt_include_tests=false')
     if cloud_gaming:
         gn_args.append('owt_cloud_gaming=true')
+        gn_args.append('enable_libaom=false')
+    else
```
Comment (dvrogozh) [inline scripts/build-win.py:86]: That's wrong syntax. You are missing `:`:
```
  File ".\\scripts\\build-win.py", line 86
    else
       ^
SyntaxError: invalid syntax
```

**Review by dvrogozh — CHANGES_REQUESTED** [review]

File: scripts/build-win.py (hunk 1 above)
Comment (jianjunz) [reply scripts/build-win.py]: Fixed. Thanks for pointing out.

Comment (dvrogozh) [reply scripts/build-win.py]: I verified fix - it works for me on Windows.

**Review by dvrogozh — APPROVED** [review]
looks good, suggest to merge

**Pull request merged**